### PR TITLE
Longitudinal causes

### PR DESCRIPTION
**Pull request by peterhurford**

### CODE REVIEW

File: data/2017/2017-longitudinal-analysis-tables.txt [hunk 1]
```diff
@@ -1031,59 +1031,71 @@ s15_cause_import_meta ### s17_cause_import_meta
    This cause is the top priority                                                   7
```
Comment (robertzk) [inline data/2017/2017-longitudinal-analysis-tables.txt]: @peterhurford What format is this in? It seems like some dump of R history statements?

File: models/2017/easurvey/easurvey.R [hunk 2]
```diff
@@ -146,6 +146,25 @@ Ramd::define("referrers", "simple_referrers", "cities",
...
+        , "AI by joining 2013 or earlier II"        = function(df) tab(df, cause_import_ai_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011", "2012", "2013"), percent = TRUE, byrow = FALSE)
+        , "AI by joining 2013 or earlier III"       = function(df) ctab(df, cause_import_ai_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011", "2012", "2013"), na.rm = TRUE)
+        , "AI by joining 2012 or earlier"           = function(df) ctab(df, cause_import_ai_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011", "2012"), na.rm = TRUE)
+        , "AI by joining 2011 or earlier"           = function(df) ctab(df, cause_import_ai_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011"), na.rm = TRUE)
+        , "AI by joining 2014 or earlier"           = function(df) ctab(df, cause_import_ai_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011", "2012", "2013", "2014"), na.rm = TRUE)
+        , "Poverty by joining 2013 or earlier"      = function(df) tab(df, cause_import_poverty_b, which_year_EA %in%  c("Before 2009", "2009", "2010", "2011", "2012", "2013"), percent = TRUE, na.rm = TRUE, byrow = FALSE)
```
Comment (robertzk) [inline models/2017/easurvey/easurvey.R]: Maybe call this (w/ NA) and the below (w/o NA) for clarity?

I assume you've thought about DRY'ing these with a grid syntax for `tab` and `ctab` but decided against it for clarity?

File: data/2017/2017-longitudinal-analysis-tables.txt (hunk 1 above)
Comment (peterhurford) [reply data/2017/2017-longitudinal-analysis-tables.txt]: It's an R print out of the tables that `surveytools2` make.

File: models/2017/easurvey/easurvey.R (hunk 2 above)
Comment (peterhurford) [reply models/2017/easurvey/easurvey.R]: The w/NA and w/o NA do appear in the print out from surveytools, though I agree that maybe the actual text could be clearer.

I'm not sure how to implement a grid here... probably would be more complicated than it's worth?

**Pull request merged**